custom/icds_reports/utils/aggregation_helpers/distributed/child_health_monthly.py

```diff
@@ -91,10 +92,13 @@ def _state_aggregation_query(self, state_id):
         alive_in_month = "(child_health.date_death IS NULL OR child_health.date_death - {} >= 0)".format(
             start_month_string
         )
-        migration_status = "(agg_migration.is_migrated=1 AND agg_migration.migration_date < {})::integer".format(
+        migration_status = "(agg_migration.is_migrated = 1 AND agg_migration.migration_date < {})::integer".format(
```

Comment:
would be good to change this to use a named parameter as well

Reply:
Done!